gotests/main.go

```diff
@@ -50,6 +53,7 @@ func main() {
 		ExportedFuncs: *exportedFuncs,
 		AllFuncs:      *allFuncs,
 		PrintInputs:   *printInputs,
+		Subtests:      *subtests,
```

Comment:
We should use the build protection markup to only include the flag in Go 1.7+. It doesn't make sense to include it in anything below. In Go 1.6 and below, we can just use a `const subtests = &false` so that this compiles. How does that sound?